### PR TITLE
Create a method to check if we should replace existing quality

### PR DESCRIPTION
**Pull request by fernandog**
I wrote the method so it could be easy do understand and to "read"

@pymedusa/developers i will run this on my branch for a few days
would be nice if someone could review this.

### CODE REVIEW

**Comment by fernandog** [comment]
@pymedusa/developers i will run this on my branch for a few days

would be nice if someone could review this.

**Comment by fernandog** [comment]
Approve or close

**Comment by fernandog** [comment]
@pymedusa/developers no one else has comments on this?

File: medusa/common.py [hunk 1]
```diff
@@ -601,6 +601,46 @@ def should_search(status, show_obj):
         return True
 
     @staticmethod
+    def should_replace(ep_status, old_quality, new_quality, allowed_qualities, preferred_qualities):
```
Comment (p0psicles) [inline medusa/common.py]: Only remark I could have, is that "old_quality, new_quality, allowed_qualities, preferred_qualities" are all similar parameters. They could be easily passed as kwargs.

**Comment by p0psicles** [comment]
Looks nice!

**Comment by fernandog** [comment]
I'm coding a more complete `should_replace` function but i will leave to another PR
also using @p0psicles suggestion

**Pull request merged**